Destination_Läjet/Destination_Lajet/Pages/Shared/_Layout.cshtml

```diff
@@ -6,6 +6,8 @@
     <title>@ViewData["Title"] - Destination_Lajet</title>
     <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
     <link rel="stylesheet" href="~/css/site.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Asap+Condensed&display=swap" rel="stylesheet">
```

Comment:
Den stora nackdelen med hrefs istället för filer är att man bygger beroende till externa resurser. Når man inte dessa av olika skäl, får man inte css:en.

Ett sätt att komma runt är att helt enkelt gå in på en länk,
https://fonts.gstatic.com eller
https://fonts.googleapis.com/css2?family=Asap+Condensed&display=swap
kopiera allt innehåll och lägga i en separat fil.

Det finns något som kan tyckas vara en drawback med detta också; man får inte uppdateringar om sådana inträffar.
Faktiskt, är det en fördel att inte få dem! Då man har kodat för en viss version av dokumentet ifråga, vill man inte att siten ska förstöras/ändras utifall att original-dokumentet skulle förändras